Gifski/util.swift

```diff
@@ -1030,7 +1030,13 @@ extension CGSize {
 
 	func aspectFit(to boundingSize: CGSize) -> CGSize {
 		let ratio = min(boundingSize.width / width, boundingSize.height / height)
-		return self * ratio
+
+		var size = self * ratio
+
+		size.width = min(self.width, size.width)
+		size.height = min(self.height, size.height)
```

Comment:
This is not correct. It doesn't preserve the aspect ratio, which is the main point of this method. It only works correctly since the `boundingSize` has the same width and height.

Reply:
Done